packages/utils/src/lib/report.unit.test.ts

```diff
@@ -102,17 +103,12 @@ describe('loadReport', () => {
   });
 
   it('should throw for an invalid JSON report', async () => {
-    const invalidReportMock = reportMock();
-    invalidReportMock.plugins = [
-      {
-        ...invalidReportMock.plugins[0],
-        slug: '-Invalud_slug',
-      } as unknown as PluginReport,
-    ];
-
     vol.fromJSON(
       {
-        [`report.json`]: JSON.stringify(invalidReportMock),
+        [`report.json`]: JSON.stringify({
+          ...REPORT_MOCK,
+          plugins: [{ ...REPORT_MOCK.plugins[0], slug: '-Invalid_slug' }],
+        }),
```

Comment:
To have autocomplete (`JSON.stringify` accepts `any`):

```suggestion
        [`report.json`]: JSON.stringify({
          ...REPORT_MOCK,
          plugins: [{ ...REPORT_MOCK.plugins[0]!, slug: '-Invalid_slug' }],
        } satisfies PluginReport),
```